concordium-contracts-common/src/types.rs

```diff
@@ -1202,6 +1202,10 @@ impl<'a> Parameter<'a> {
     /// external means.
     #[inline]
     pub fn new_unchecked(bytes: &'a [u8]) -> Self { Self(bytes) }
+
+    /// Construct an empty parameter.
+    #[inline]
+    pub fn empty() -> Self { Self::default() }
```

Comment:
I'd prefer if we constructed it directly here.

It would not be unreasonable to implement default in terms of empty. Which would lead to an infinite loop.